MapboxNavigation/FeedbackCollectionViewCell.swift

```diff
@@ -0,0 +1,128 @@
+import Foundation
+import UIKit
+
+
+class FeedbackCollectionViewCell: UICollectionViewCell {
+    static let defaultIdentifier = "MapboxFeedbackCell"
+    
+    struct Constants {
+        static let imageSize: CGSize = 70.0
+        static let circleLabelSpacing: CGFloat = 8.0
+    }
+    
+    lazy var imageView: UIImageView = {
+        let view: UIImageView = .forAutoLayout()
+        return view
+    }()
+    
+    lazy var titleLabel: UILabel = {
+        let title: UILabel = .forAutoLayout()
+        title.numberOfLines = 2
+        title.textAlignment = .center
+        title.font = .systemFont(ofSize: 18.0)
+        return title
+    }()
+    lazy var circleView: UIView = .forAutoLayout()
+    
+    lazy var circleViewWidthConstraint = {
```

Comment:
Lazy loading the views makes it easier to digest as it isolates the relevant parts, but a lazy var for each of these constant constraints is excessive. It's arguably harder to read, and it increases the loc count by a factor of three for no reason. Can we inline these in `setupConstraints()`?